test/run_test.py

```diff
@@ -264,7 +264,7 @@
     'test_jit',
     'test_jit_profiling',
     'test_torch',
-    'test_binary_ufuncs'
+    'test_binary_ufuncs',
```

Comment:
Without this comma, the strings are concatenated to `test_binary_ufuncstest_numpy_interop`.

Reply:
yikes, nice fix